### PR TITLE
TOF: PID plots per track source and event time 

### PR DESCRIPTION
**Pull request by ercolessi**
@noferini @njacazio

### CODE REVIEW

File: Modules/PID/src/TaskFT0TOF.cxx [hunk 1]
```diff
@@ -50,23 +50,27 @@ namespace o2::quality_control_modules::pid
 
 bool MyFilter(const MyTrack& tr)
 {
-  return (tr.getP() < 2.0);
+  return (tr.getP() < 1.5);
```
Comment (njacazio) [inline Modules/PID/src/TaskFT0TOF.cxx:53]: Maybe we want this configurable without having to change the code?

**Review by njacazio — COMMENTED** [review]
Ciao Francesca, huge work, thanks!

File: Modules/PID/src/TaskFT0TOF.cxx (hunk 1 above)
Comment (ercolessi) [reply Modules/PID/src/TaskFT0TOF.cxx]: Ciao @njacazio indeed thanks for spotting this! I changed it to configurable

**Review by njacazio — APPROVED** [review]

File: Modules/PID/src/TaskFT0TOF.cxx (hunk 1 above)
Comment (noferini) [reply Modules/PID/src/TaskFT0TOF.cxx]: I guess that
float t0maxp = 1.5;
should be 
static float t0maxp;
(the same for the getter)

Comment (ercolessi) [reply Modules/PID/src/TaskFT0TOF.cxx]: Ciao @noferini thanks a lot for spotting this I changed it to static!

**Review by njacazio — APPROVED** [review]

**Review by njacazio — APPROVED** [review]

**Review by njacazio — APPROVED** [review]

**Review by Barthelemy — APPROVED** [review]

**Pull request merged**